src/actions/generic/index.ts

```diff
@@ -52,6 +55,37 @@ export const initApp: ThunkAction = async (
   }
 }
 
+const hashString = (text: string) => {
+  return crypto
+    .createHash('sha256')
+    .update(text)
+    .digest('hex')
+}
+
+export const checkTermsOfService = (
+  route: routeList,
+): ThunkAction => async dispatch => {
+  const storageHash = await AsyncStorage.getItem('termsOfConditions')
```

Comment:
I think we can use const for termsOfConditions to reuse it and avoid errors because of typos